compiler/rustc_target/src/target_features.rs

```diff
@@ -34,6 +34,9 @@ pub enum Stability {
     /// particular for features are actually ABI configuration flags (not all targets are as nice as
     /// RISC-V and have an explicit way to set the ABI separate from target features).
     Forbidden { reason: &'static str },
+    /// This feature can not be set via `-Ctarget-feature` or `#[target_feature]`, it can only be set
+    /// by target modifier flag. Target modifier flags are tracked to be consistent in linked modules.
+    TargetModifierOnly { reason: &'static str, flag: &'static str },
```

Comment:
Why are such features even listed here? The entire point of this file is to define the Rust feature names used for `-Ctarget-feature` and `#[target_feature]` and `cfg(target_feature)`. Mixing in other things here seems like a big mess...